client/src/components/reviewsSubComponents/RatingStars.jsx

```diff
@@ -28,26 +30,95 @@ const StarsContainer = styled.div`
     display: block;
     z-index: 0;
   }
+
+`;
+
+const InputStarWrapper = styled.div`
+  fontSize: 25px;
+  color: #FDD700;
+  cursor: pointer;
 `;
 
-const RatingStars = (props) => (
-  <StarsContainer>
-    <StarsContainer className="top" percent={props.percent}>
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-    </StarsContainer>
-    <StarsContainer className="bottom">
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-      <span>★</span>
-    </StarsContainer>
-  </StarsContainer>
-
-);
+class RatingStars extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedStars: [1, 2, 3, 4, 5],
+      rating: this.props.rating || 0,
+      hovered: 0,
+      selectedIcon: '★',
+      deselectedIcon: '☆',
```

Comment:
You should use these for the z-index: 0 stars in the other star rating component, too. This functionality looks really good